dashboard/modules/metrics/metrics_head.py

```diff
@@ -136,7 +138,8 @@ async def get_progress(self, req):
         """
         job_id = req.query.get("job_id")
         job_id_query = f'{{JobId="{job_id}"}}' if job_id else ""
-        query = f"sum(ray_tasks{job_id_query}) by (State)"
+        # Fetches the max_over_time over last 14 days
+        query = f"sum(max_over_time(ray_tasks{job_id_query}[14d])) by (State)"
```

Comment:
I think you only want this for FINISHED and FAILED states, otherwise you may double count non-terminal states.